components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -760,6 +772,39 @@ void HommeDynamics::homme_post_process () {
       V_prev(0,ilev) = v(0,ilev);
       V_prev(1,ilev) = v(1,ilev);
     });
+
+    // Calculate decay rate profile, otau,
+    // and apply Rayleigh friction
+    Real krange; // range of rayleigh friction profile
+    Real tau0;   // approximate value of decay time at model top
+    Real otau0;  // inverse of tau0
+
+    krange = raykrange;
+    if (raykrange == 0) krange = (rayk0 - 1.0)/2.0;
+
+    tau0 = 86400.0*raytau0; // convert to seconds
+    otau0 = 0;
+    if (tau0 != 0) otau0 = 1/tau0;
+
+    if (otau0 != 0) {
+      auto ws = wsm.get_workspace(team);
+      ekat::Unmanaged<WS::view_1d<Pack> > otau = ws.take("otau");
+
+      Kokkos::parallel_for(Kokkos::TeamThreadRange(team, npacks),
```

Comment:
One rank could store a single 1D view init'ed once at init time.

Reply:
Alternatively, since this is our impl, the RF impl could just compute otau during its pass over the data and no extra view input would be needed.

Reply:
Separately, please make sure to verify your otau profile matches the EAM one.

Reply:
Yes, I verified the profiles matched. I may just change the `PF::apply_rayleigh_friction()` to compute the `otau` vals.

Reply:
Actually, do you have a preference: 1d view in the mem buffer vs. internally computing profile? Adding to the mem buffer closer resembles eam.

Reply:
Either is fine, but if you keep it as a view, I recommend a single 1D view per rank computed at init, since otau is independent of dynamic and spatially varying quantities.